css/class-naming-conventions/readme.md

```diff
@@ -89,15 +89,23 @@ Like components these should always live at the root level of a file. Do not nes
     .c-blog-post__title {
     }
 }
+
+// Bad!
+.c-blog-post__title__emote {
+}
+
+// Good!
+.c-blog-post__emote {
+}
 ```
 
 ### Modifiers
 
-These are used to modify components or subcomponents. They are always chained to a specific component and are declared in the component or subcomponent that they affect.
+Modifiers, as their name suggests, modify components or sub-components. They are always chained to the component or sub-component they belong to.
 
 * Prefixed with the namespace of the affected element and two dashes (`c--`, `t--`)
 * Contained to the scope of a single component
-* Always declared as a chained selector to a component or subcomponent.
+* Always declared as a chained selector to a component or sub-component.
 * Never declared as a stand-alone rule.
```

Comment:
Should we link to the *Modifiers that Affect Sub-components* section of the doc here?